### PR TITLE
[metadata] Metadata fixes

### PR DESCRIPTION
**Pull request by wrwg**
Measures metadata size

### CODE REVIEW

**Review by runtian-zhou — APPROVED** [review]

File: aptos-move/aptos-vm/src/aptos_vm_impl.rs [hunk 1]
```diff
@@ -58,6 +58,7 @@ pub struct AptosVMImpl {
     storage_gas_params: Result<StorageGasParameters, String>,
     version: Option<Version>,
     features: Features,
+    timed_features: TimedFeatures,
```
Comment (zekun000) [inline aptos-move/aptos-vm/src/aptos_vm_impl.rs:61]: unrelated change?

Comment (wrwg) [reply aptos-move/aptos-vm/src/aptos_vm_impl.rs:61]: This is needed in the other repo and will help also in the future, so lets keep it.

**Review by gedigi — APPROVED** [review]
